www/include/core/header/htmlHeader.php

```diff
@@ -126,7 +126,11 @@
         $searchStr .= "search_host=" . htmlentities($_GET["search"], ENT_QUOTES, "UTF-8");
     }
     if (isset($centreon->historySearch[$url]) && !isset($_GET["search"])) {
-        $searchStr .= "search_host=" . $centreon->historySearch[$url];
+        $searchStr .= "search_host=" . (
+                is_array($centreon->historySearch[$url]) && isset($centreon->historySearch[$url]['searchH']) ?
+                    $centreon->historySearch[$url]['searchH'] :
+                    $centreon->historySearch[$url]
```

Comment:
historySearch is an array of arrays. So (in the else case of your ternary), you're affecting an array to "search_host=" (line 129), and it won't work

Reply:
the old code was "$searchStr .= "search_host=" . $centreon->historySearch[$url];" which produced PHP warning.

Reply:
if : "isset($centreon->historySearch[$url]['searchH']" isn't set -> you send the whole array $centreon->historySearch[$url] ?? this won't work